### PR TITLE
Symantec mss

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20171

## Description
-Used xml character escaping in `comments`.
-No tests were added on purpose, was sent for customer validation.

### CODE REVIEW

**Comment by teizenman** [comment]
@roysagi Please verify RN

File: Integrations/SymantecMSS/SymantecMSS.yml [hunk 1]
```diff
@@ -1,99 +1,151 @@
+category: Case Management
```
Comment (ronykoz) [inline Integrations/SymantecMSS/SymantecMSS.yml]: What was changed in this file? Seems like a lot just changed it’s place

Comment (teizenman) [reply Integrations/SymantecMSS/SymantecMSS.yml]: Thats exactly what happened, the PyCharm plugin did the manipulations, but no change was done.

**Comment by ronykoz** [comment]
@teizenman we can merge this

**Pull request merged**